docs/api_docs/python.rst

```diff
@@ -0,0 +1,170 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _getting_started_python_api:
+
+******************************
+Getting Started with Python
+******************************
+
+Installation
+==================
+.. note::
+    We heavily recommend to manage your projects with some sort of virtual environment, for example Pipenv. For instructions to install Pipenv for Mac or Windows click `here <https://medium.com/@mahmudahsan/how-to-use-python-pipenv-in-mac-and-windows-1c6dc87b403e>`_. For installation instructions for Linux click `here <https://github.com/pypa/pipenv>`_.
```

Comment:
This may be a bit outside of our purview to recommend the usage of virtual environments to people. May want to restrict ourselves to just things specific to our library.  So you may want to lose this note and the installing with Pipenv part below. 

I know it previously existed on our old Getting Started with Python, but honestly I don't see the point in us recommending virtual environments to people. We are just another library, let people manage their environments anyway they like. It makes no difference to us.